src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -41,11 +41,13 @@ class UnifiedShareModal extends React.Component<USMProps, State> {
     constructor(props: USMProps) {
         super(props);
 
+        const { initialDataReceived } = props;
+
         this.state = {
-            getInitialDataCalled: false,
+            getInitialDataCalled: !!initialDataReceived,
```

Comment:
Is there a way to infer getInitialDataCalled based on the data available?

Reply:
I originally tried to use `!!item && !!sharedLink`, but that ended up changing a bunch of USM snapshot tests. `componentDidUpdate()` in the USM also checks for `getInitialDataCalled`, `type`, and `typeID`, rather than `item`/`sharedLink`. I'm not sure whether there's a case in which we'd still want to call `getInitialData` even when `item`/`sharedLink` are defined, but I figured it would be best for backwards-compatibility to add a new prop that doesn't affect existing instantiations of the USM.